<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="plotGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="plotGraphicsView_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="player1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="curr_player">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="coordinates_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter coordinates</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coordinates_entered">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="to_shoot">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>to shoot:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ships_view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ships_view_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Place selected ship starting at:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="place_ship_buttton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Place</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ship_input_x">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_Game">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>397</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit_Game">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ship_input_y">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBattleship">
    <property name="title">
     <string>Battleship</string>
    </property>
   </widget>
   <addaction name="menuBattleship"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
